apps/studio/components/interfaces/Settings/Logs/LogsPreviewer.tsx

```diff
@@ -239,23 +225,28 @@ export const LogsPreviewer = ({
       <div
         className={
           'transition-all duration-500 ' +
-          (showChart && logData.length > 0 ? 'mb-4 h-28 opacity-100' : 'h-0 opacity-0')
+          (showChart && logData.length > 0 ? 'mb-2 mt-1 opacity-100' : 'h-0 opacity-0')
         }
       >
         <div className={condensedLayout ? 'px-3' : ''}>
           {showChart && (
-            <LogEventChart
-              className={cn({
-                'opacity-40': isLoading,
-              })}
+            <LogsBarChart
               data={eventChartData}
-              onBarClick={(isoTimestamp) => {
+              onBarClick={(datum) => {
+                if (!datum?.timestamp) return
                 handleSearch('event-chart-bar-click', {
                   query: filters.search_query as string,
-                  to: isoTimestamp as string,
+                  to: datum.timestamp.toString(),
                   from: null,
                 })
               }}
+              EmptyState={
```

Comment:
just to check - when should this show up? I tried on my staging project for storage logs, there's no data but it shows this UI instead

![Uploading image.png…]()